### ISSUE TITLE
Report selected text silent when large amount of text selected

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#9214]
### Steps to reproduce:

1. Find a large amount of text - I selected the entire WW1 article from Wikipedia and copied it into Notepad (just to be sure it was nothing to do with the browser).
2. Select a large amount of text (I selected all 359,044 characters).
3. Press NVDA+shift+up arrow (NVDA+shift+s in laptop keyboard layout).

### Actual behavior:

NVDA is silent.  NVDA responds if another command is given (reading the time, pressing alt+tab etc).

The log contains the following:
```
IO - inputCore.InputManager.executeGesture (11:53:14.095):
Input: kb(desktop):shift+NVDA+upArrow
DEBUG - synthDrivers.oneCore.SynthDriver.cancel (11:53:14.117):
Cancelling
IO - speech.speak (11:53:14.127):
Speaking [LangChangeCommand (u'en_GB'), u'Selected 
...
(Entirety of selected text inserted in log)
...
DEBUG - synthDrivers.oneCore.SynthDriver._processQueue (11:53:16.973):
Begin processing speech
ERROR - NVDAHelperLocal (11:55:13.559):
Thread 21464, build\x86\localWin10\oneCoreSpeech.cpp, ocSpeech_speak::<lambda_9a3b2dc9b65af643cf9164764bf543e6>::operator (), 85:
Error -2147024882: Not enough memory resources are available to complete this operation.



Not enough memory resources are available to complete this operation.



DEBUGWARNING - synthDrivers.oneCore.SynthDriver._callback (11:55:13.559):
ocSpeech_speak failed!
```

### Expected behavior:

NVDA should read the selected text OR report an error or read the number of characters selected as it does when selecting text.

#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
2018.4.1
#### Windows version:
Windows 10 Version: 
18272.1000 
#### Does the issue still occur after restarting your PC?
Yes
#### Have you tried any other versions of NVDA?
No.

**Comment by Brian1Gaff** [comment]
So does this still happen on Espeak?
 
Do we have here a problem about memory limits on 32 bit programs?
Brian

bglists@blueyonder.co.uk 
Sent via blueyonder. 
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
-----

**Comment by Qchristensen** [comment]
I just tried recreating the same situation using eSpeak or SAPI 5 and both read the text (with about a five second pause between pressing NVDA+shift+up arrow and NVDA starting to read.

**Comment by Brian1Gaff** [comment]
Yes I'd known about the delay, but using windows 7 as I do I of course 
cannot use the one core voices.
 I do a lot of cutting and pasting from web pages, emails and documents 
generally to edit copy for our tn and have never found a lock up, just a 
pregnant pause.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
-----
